src/schema/rules/sidecars/motion.yaml

```diff
@@ -0,0 +1,74 @@
+#
+# Groups of related metadata fields
+#
+# Assumptions: never need disjunction of selectors
+# Assumptions: top-to-bottom overrides is sufficient logic
+
+---
+motionGeneric:
+  selectors:
+    - modality == "motion"
+    - datatype == "motion"
+    - suffix == "motion"
+  fields:
+    TaskName:
+      level: required
+
+motionRecommended:
+  selectors:
+    - modality == "motion"
+    - datatype == "motion"
+    - suffix == "motion"
+  fields:
+    InstitutionName: recommended
+    InstitutionAddress: recommended
+    InstitutionalDepartmentName: recommended
+    Manufacturer: recommended
+    ManufacturersModelName: recommended
+    SoftwareVersions: recommended
+    TaskDescription: recommended
+    Instructions: recommended
+    DeviceSerialNumber: optional
+    ExternalSoftwareVersions: optional
```

Comment:
CI says this is not defined in `src/schema/objects/metadata.yaml`.

https://github.com/bids-standard/bids-specification/actions/runs/4127444754/jobs/7130662091#step:5:106

Reply:
temporarily commented to pass validation

Reply:
Cleaned up the sidecar with current keywords from BIDS